src/app.ts

```diff
@@ -1,9 +1,16 @@
 import express from "express";
+import mongoose from "mongoose";
+import bodyParser from "body-parser";
 import swaggerUi from "swagger-ui-express";
 import swaggerJSDoc from "swagger-jsdoc";
+import movieRoutes from "./routes/movieRoutes";
+import genreRoutes from "./routes/genreRoutes";
 
 const app = express();
 const port = 3000;
+app.use(bodyParser.json());
+
+mongoose.connect("mongodb://localhost:27017/ExampleDB");
```

Comment:
It is not a secure way to save such links in repos. Even if it ways to localhost. Better to store it in `.env` file and load from it. .env file should be excluded from committing to git in `.gitignore` file.

Reply:
Have added .env file and moved there link to data base.